### PR TITLE
Create the base playbook

### PR DESCRIPTION
**Pull request by pachr**
We are going to create a playbook to provision a new server with basic packages, setup users with ssh keys, setup git.

#### 1. Create the playbook `base.yml`
  - [x] 1.1 remote_user will be itmarkets
  - [x] 1.2 pass the git key
  - [x] 1.3 call the base role

#### 2. Create the base role
  - [x] 2.1 make sure devops group is present
  - [x] 2.2 Verify that ubuntu and itmarkets are present
  - [x] 2.3 check git access
    - [x] 2.3.1 copy the private key to the server
  - [x] 2.4 Install base packages
  - [x] 2.5 Install Python packages
  - [x] 2.6 Create a new MOTD on an ascii art generator
  - [x] 2.7 in the script update the default MOTD
  - [x] 2.8 Set the localtime to UTC with symlink

#### 3. Create inventories (dev, staging, production), they will be empty.
  - [x] 3.1 done?

#### 4. Dependencies

- [x] 4.1 We need a new git key
- [x] 4.2 What will be the name of the user to build? (itmarkets?)
- [x] 4.3 Create ssh key for this user
- [x] 4.4 Create ansible-vault password

### CODE REVIEW

**Comment by pachr** [comment]
@remitaffin You will find a list of dependencies at the end of the description. Could you provide me them or answer my questions? Please

**Comment by remitaffin** [comment]
Yes, the new user will be itmarkets. No other users for now.
We have the ssh keys for this user. I'll send them to you.
You can create the git ssh key as we didn't create it yet. Then we'll add the public key to the github account.

**Comment by pachr** [comment]
@remitaffin For this pull request, just the git ssh key is missing. When I will have it, I will encrypt the private key by ansible-vault in `env_vars/base/git_key.yml`

**Comment by darwinyc** [comment]
We don't do this now? 

Darwin Abella
VP of IT | The Noodle Companies, LLC

**Comment by pachr** [comment]
@darwinyc I've created the ssh keys. But the git ssh key should be tied to the user "itmarkets", and I don't have access to that.

**Comment by darwinyc** [comment]
Sorry. Let's talk about this next week. You shouldn't be creating keys because we already created them. @remitaffin, the scope needs to be clear. 

Darwin Abella
VP of IT | The Noodle Companies, LLC

**Comment by pachr** [comment]
@remitaffin I added the private git key and encrypted the file with ansible-vault. Could you review this code when you have time? please

File: ansible/roles/base/tasks/reset_users.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- name: Verify users presence
+  user: name=itmarkets state=present groups=devops
+
+- name: Verify .ssh directory in home
+  file: path=/home/itmarkets/.ssh/ state=directory owner=itmarkets group=itmarkets mode=0700
+
+- name: Verify ssh public key to authorized_keys
+  template: src=itmarkets.pub dest=/home/itmarkets/ owner=itmarkets group=itmarkets mode=0600
+
+- name: Verify that ubuntu and itpartners users are still present
```
Comment (remitaffin) [inline ansible/roles/base/tasks/reset_users.yml]: itpartners?

File: ansible/roles/base/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  tags: packages
+
+- name: Install Python packages
+  pip: name={{ item }}
+  with_items:
+    - watchdog
+    - boto
+  tags: packages
+
+- name: Update the MOTD with partners template
```
Comment (remitaffin) [inline ansible/roles/base/tasks/main.yml]: Partners??

**Comment by remitaffin** [comment]
Good to merge now! :+1:

File: ansible/roles/base/tasks/main.yml (hunk 2 above)
Comment (pachr) [inline ansible/roles/base/tasks/main.yml]: I fixed it.

File: ansible/roles/base/tasks/reset_users.yml (hunk 1 above)
Comment (pachr) [inline ansible/roles/base/tasks/reset_users.yml]: I fixed it.

**Comment by pachr** [comment]
Ok I'll do it right now! Do you want me to merge to master or create dev and stage branch?

**Comment by remitaffin** [comment]
There's no dev or stage branches on build or partnersbuild.
Let's keep it consistent.
Please merge to master.

**Comment by pachr** [comment]
it's merged

**Pull request merged**